.github/workflows/prestocpp-worker-with-clp-connector-runtime-image-build.yml

```diff
@@ -0,0 +1,97 @@
+name: prestocpp-worker-with-clp-connector-runtime-image-build
+
+on:
+  # TODO: specifiy the branch to the release-0.293 when finalize the PR
+  pull_request:
+  push:
+
+jobs:
+  prestocpp-worker-with-clp-connector-runtime-image:
+    name: prestocpp-worker-with-clp-connector-runtime-image
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: "Login to Image Registry"
+        uses: "docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772"
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: "Update Metadata for Dependency Image"
```

Comment:
Can we reorder these steps so the order is:

* Set up metadata for deps image
* Build deps image
* Set up metadata for runtime image
* Build runtime image